base/docs/bindings.jl

```diff
@@ -14,7 +14,9 @@ struct Binding
     end
 end
 
-bindingexpr(x) = Expr(:call, Binding, splitexpr(x)...)
+binding(m::Module, v::Symbol) = Binding(m, v)
+binding(x, v::Union{AbstractString,Symbol}) = getproperty(x, v)
```

Comment:
Why not just `binding(x, v) = getproperty(x, v)`?

Reply:
Ah yes, that's better. I changed it in the new commit.